README.md

```diff
@@ -1,69 +1,105 @@
+These best practice Jenkinsfiles are based on the general concepts of [Pipelines
+as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the
+Jenkins community at hand to optimize your continuous delivery processes.
 
-You can run the Jenkinsfiles out of the box or take them as a starting point for project-specific adaptations.
+You can run the Jenkinsfiles out of the box or take them as a starting point for
+project-specific adaptations.
 
-## Installation
+# Requirements
 
-Prerequisites:
-
-* [Installed shared library][piper-library-installation] of project "Piper".
+* [Installed shared library][piper-library-readme] of project "Piper".
 * Jenkins User with privileges to create jobs.
 
-There are two recommended ways to consume our `Jenkinsfile`. For both ways, you need to create a new pipeline job in Jenkins that [retrieves a pipeline from an SCM][jenkins-doc-pipelineFromSCM].
-The location of the `Jenkinsfile` may differ:
+# Download and Installation
+
+There are two recommended ways to consume our `Jenkinsfile`. For both ways, you
+need to create a new pipeline job in Jenkins that [retrieves a pipeline from an
+SCM][jenkins-doc-pipelineFromSCM].  The location of the `Jenkinsfile` may
+differ:
 
-1. The `Jenkinsfile` can be copied into the source code repository of the application. In that case, no additional pipeline job parameters are required to retrieve the application source code repository.
+1. The `Jenkinsfile` can be copied into the source code repository of the
+   application. In that case, no additional pipeline job parameters are required
+   to retrieve the application source code repository.
 
-2. The `Jenkinsfile` can be kept in an own repository separate from the application. In case you want to apply the `Jenkinsfile` to multiple applications, this approach is beneficial since you only need to maintain the `Jenkinsfile` in one place. You must define two mandatory Jenkins pipeline job parameters in this case:
+2. The `Jenkinsfile` can be kept in an own repository separate from the
+   application. In case you want to apply the `Jenkinsfile` to multiple
+   applications, this approach is beneficial since you only need to maintain the
+   `Jenkinsfile` in one place. You can consume the `Jenkinsfile` e.g. directly from
+   this repository on GitHub.com or from your own fork of it. You must define two
+   mandatory Jenkins pipeline job parameters to define the location of your
+   application:
 
     * `GIT_URL`: The URL to the Git repository in which the application resides.
-    * `GIT_BRANCH`: The branch of the Git repository that should be checked out for the built.
+    * `GIT_BRANCH`: The branch of the Git repository that should be checked out
+      for the build.
+
+# Configuration
```

Comment:
Maybe a list of a few such files with a link to the file?

Reply:
A few Jenkinsfiles you mean? There is a list in the subsequent subsection of this section.

Reply:
No. I was thinking of links to the template configuration files, such as 'Gruntfile', 'mta.yaml', 'package.json' etc.
Although I admit it's written in the text:
> Templates for these configuration files are available in the same folders as the respective `Jenkinsfile`.